MapboxVision/src/main/java/com/mapbox/vision/VisionManager.kt

```diff
@@ -257,6 +293,10 @@ object VisionManager : BaseVisionManager {
         attachableModules.forEach { it.attach() }
     }
 
+    override fun addObservable(observer: VisionEventsListener) = delegate.addObservable(observer)
+
+    override fun removeObserver(observer: VisionEventsListener) = delegate.removeObserver(observer)
```

Comment:
We really should discuss if we can make this functions public. Lets discuss this once more with @deniskoronchik and @mapbox/vision-ios

Reply:
discussed